app/patients/edit/controller.js

```diff
@@ -517,6 +517,12 @@ export default AbstractEditController.extend(BloodTypes, ReturnTo, UserSession,
   },
 
   afterUpdate: function(record) {
+    var patient = this.get('model');
+    if (patient.getProperties('createOutpatientVisit').createOutpatientVisit) {
```

Comment:
instead of
 ```javascript
if (patient.getProperties('createOutpatientVisit').createOutpatientVisit) {
```
You should use patient.get. For example:
 ```javascript
if (patient.get('createOutpatientVisit')) {
```